samples/php8/ecom-server/public/js/sdk-common.php

```diff
@@ -0,0 +1,73 @@
+<?php
+header('Content-Type: application/javascript');
+?>
```

Comment:
Use php to return js first, as our phpfpm not support js files.
I will update the infra later